cob_description/urdf/cob4_arm/cob4_arm.gazebo.xacro

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="cob4_arm_gazebo" params="name">
+
+<!--
```

Comment:
was this commented in schunk_description too? why?
if uncommenting is neccessary: please remove